digital-credentials/support/helper.js

```diff
@@ -54,69 +57,64 @@ const allMappings = {
  * Routes calls from specific public functions.
  * @private
  * @param {'get' | 'create'} type - The type of operation.
- * @param {string | string[]} [requestsToUse] - Raw input for request types from public function.
- * @param {string} mediation - Mediation requirement (default handled by public function).
- * @returns {{ digital: { requests: any[] }, mediation: string }}
+ * @param {string | string[]} protocol - Protocol(s) to use.
+ * @param {CredentialMediationRequirement} mediation - Mediation requirement.
+ * @returns {{ digital: { requests: any[] }, mediation: CredentialMediationRequirement }}
  * @throws {Error} If type is invalid internally, or input strings are invalid.
  */
-function _makeOptionsUnified(type, requestsToUse, mediation) {
+function _makeOptionsUnified(type, protocol, mediation) {
```

Comment:
Optional: Since we are renaming this, may be it is more clear to indicate it can be or more protocols?
```suggestion
function _makeOptionsUnified(type, protocolOrProtocols, mediation) {
```

Reply:
Ok, will rename as a followup.... we could also force the array now, as no test now passes anything but the array.